clang/include/indexstore/IndexStoreCXX.h

```diff
@@ -101,10 +102,15 @@ class IndexStore {
   friend class IndexUnitReader;
 
 public:
-  IndexStore(StringRef path, std::string &error) {
+  IndexStore(StringRef path, clang::PathRemapper remapper, std::string &error) {
```

Comment:
Same question as in clang/include/clang/Index/IndexDataStore.h, though this one really does look like it should be a const ref.